source-code/core2/src/fullExample.ts

```diff
@@ -74,24 +71,42 @@ const inlang = createInlang({
 })
 
 // --- CRUD ---
-inlang.messages.query.create({ id: "myMessageId", languageTag: "en", pattern: [{"type": "Text", value:  "Hello World" }]})
+inlang.messages.query.create({
+	data: {
```

Comment:
why the `data` prop?

Reply:
inspired by https://www.prisma.io/docs/concepts/components/prisma-client/crud#update